### ISSUE TITLE
Paperback, JAWS and reading with a Braille display

### ISSUE THREAD

**Issue by irrah68** [trypsynth/paperback#199]
There seems to be a problem in reading long paragraphs with JAWS when using a Braille display or with speech when using the JAWS cursor or the invisible cursor. It doesn't matter whether I try to read an epub file, a text file or what ever supported file format, when the text of a file consists of long lines, paragraphs. Say all function of JAWS (Insert + Down arrow) works correctly, all the paragraphs are read as they should be read, but when I use my Braille display's navigation keys to scroll forward, the endings of every long paragraph are cut off.

This problem can also be reproduced by pressing the JAWS command Insert + b.

Can anyone here reproduce this problem? If you have JAWS, you can use the book mentioned in issue #198 as a test material.

When I try to read this book with my Braille display, I get this kind of text.

There was no reason why I shouldn't have been sent for the beer that day, for the last ends of the Fairmont National Bank case had been gathered in the week before and there was nothing for me to do but errands, and Wolfe never hesitated about running me down to Murray Street for a can of shoe-polis
Wolfe lifted his head.
[...]

I have many epubs, but they are written in Finnish, so this Nero Wolfe book is a good example for testing.

I guess that this is a very complex problem because it can be seen also in QRead. It is likely to be more a JAWS problem than a problem in Paperback, but I don't know how to solve it. It is neeldess to write to Freedom Scientific, because these kinds of open source programs are not on their priority lists.

Any ideas? I have tried various JAWS settings, but nothing has changed yet.

**Comment by trypsynth** [comment]
People have reported this before, and I think it's out of my hands unfortunately. As you mentioned, it also happens with QRead and JAWS, and especially given Glen Gordon and Joe Stephen recently left Vispero and JAWS seems to rapidly be inshitifying, the likelyhood of them doing anything about this is pretty low. I'll ask Brian Hartgen about this though, he's by far one of the most talented JAWS scripters I know.

**Comment by trypsynth** [comment]
Time to get very loud and annoying, I guess. My contact at Vispero went completely silent when I tried to follow up on this issue. If you're effected by this problem, or just feel like sending a support email to Vispero to help me out, I'd much appreciate it!
